### DISCUSSION TITLE
[offload] CLI/Rest endpoint to list and describe offloaded topics of namespace

### DISCUSSION THREAD

**Discussion by shiv4289** [apache/pulsar#18839] [General]
**Is your feature request related to a problem? Please describe.**
Currently, it is difficult to find which topics of a namespace are offloaded to secondary storage. It would be have such a command / REST end point. There should be other related information (like size, time info, link?) against this list.

The one alternative as of today is the prometheus info where a query of the following nature could throw some insights..

> pulsar_storage_offloaded_size != 0

**Comment by ivan970101** [comment]
I'm interested in this issue. May I have a try?

**Comment by shiv4289** [comment]
sure @ivan970101. Please go ahead. I would be happy to be work as a reviewer.
